tao_compiler/mlir/disc/transforms/disc_specialize_fusion_with_speculation.cc

```diff
@@ -555,6 +555,16 @@ struct DiscSpecializeFusionWithSpeculationPass
       return;
     }
 
+    if (isMemIntensiveOptExperimentalEnabled()) {
+      // skip if the root is concatenate operator
+      FusionPatternBase fusion_pattern(fusion_op);
+      for (auto root_op : fusion_pattern.getRootOps()) {
+        if (isa<lmhlo::ConcatenateOp>(root_op)) {
+          return ;
```

Comment:
Looks like an extra space before `;`. Have you run git-clang-format before the commit?

Reply:
No. It is done now